src/main/java/net/coderbot/iris/block_rendering/BlockRenderingSettings.java

```diff
@@ -7,6 +7,7 @@
 
 	private boolean reloadRequired;
 	private IdMap idMap;
+	private float ambientOcclusionLevel;
```

Comment:
This should always be initialized to 1.0

Reply:
You might have missed this

Reply:
This is initialized to 1.0 in the constructor?

Reply:
It's better to give it a default value here instead of just assuming that a constructor will be called and that it will set the value properly